### ISSUE TITLE
AAPT2 - Compilation fail due to teads integer

### ISSUE THREAD

**Issue by Jems22** [teads/TeadsSDK-android#93]
Hi, 

With the new version of aapt, released with AndroidStudio 3 , the application build failed with the following error

```
Error:java.lang.RuntimeException: java.lang.RuntimeException: com.android.builder.internal.aapt.AaptException: AAPT2 compile failed:
Error:Execution failed for task :apk:mergeReleaseResources.
> Error: java.lang.RuntimeException: java.lang.RuntimeException: com.android.builder.internal.aapt.AaptException: AAPT2 compile failed:
  aapt2 compile -o ./project/apk/build/intermediates/res/merged/release ./project/apk/build/intermediates/incremental/mergeReleaseResources/merged.dir/values/values.xml
  Issues:
   - ERROR: ./project/apk/build/intermediates/incremental/mergeReleaseResources/merged.dir/values/values.xml:929 invalid integer
   - ERROR: ./project/apk/build/intermediates/incremental/mergeReleaseResources/merged.dir/values/values.xml:930 invalid integer
```

Below, the associated lines in the values.xml file :

```
    <integer name="teads_activity_transition_fast">  300 </integer>
    <integer name="teads_activity_transition_normal">500 </integer>
```

Can you please simply remove the spaced around the int values to fix the compilation issue ?

Thanks.

**Comment by benjaminvolland** [comment]
Hi @Jems22 ,
Thank you for the report. It will be done in the next release

**Comment by Jems22** [comment]
Ok thanks.
Do you have any planned date for this release ?

**Comment by benjaminvolland** [comment]
Hi @Jems22 , It's now available in the version 2.4.11
